build_library/sysext_prod_builder

```diff
@@ -0,0 +1,154 @@
+    pkginfo_opt="--base_pkginfo=${output_dir}/${pkginfo}"
+  fi
+
+  # Include FS mangle script if present
+  if [[ -x "${BUILD_LIBRARY_DIR}/sysext_mangle_${name}" ]] ; then
+    manglefs_opt="--manglefs_script=${BUILD_LIBRARY_DIR}/sysext_mangle_${name}"
+    msg="${msg}, FS mangle script 'sysext_mangle_${name}'"
+  fi
+
+  info "${msg}."
+  
+  sudo "${SCRIPTS_DIR}/build_sysext" \
+            --board="${BOARD}" \
+            --image_builddir="${workdir}/sysext-build" \
+            --squashfs_base="${base_sysext}" \
+            --generate_pkginfo \
+            ${manglefs_opt} ${pkginfo_opt} \
+            "${name}" "${grp_pkg}"
+
+  sudo mv "${workdir}/sysext-build/${name}.raw" "${workdir}/sysext-build/${name}_pkginfo.raw" \
+                              "${workdir}/sysext-build/${name}"_*.txt "${output_dir}"
+
+  sudo mkdir -p "${install_root}"/usr/share/flatcar/sysext
+  sudo install -m 0644 -D "${output_dir}/${name}.raw" "${install_root}"/usr/share/flatcar/sysext/
+
+  sudo mkdir -p "${install_root}"/etc/extensions/
+  sudo ln -sf "/usr/share/flatcar/sysext/${name}.raw" "${install_root}/etc/extensions/${name}.raw"
+}
+# --
+
+BOARD="$1"
+BUILD_DIR="$2"
+root_fs_dir="$3"
+
+merged_rootfs_dir="$4"
+sysext_output_dir="$5"
+
+sysexts_list="$6"
+
+grp_pkg=""
+prev_pkginfo=""
+sysext_workdir="${BUILD_DIR}/prod-sysext-work"
+sysext_mountdir="${BUILD_DIR}/prod-sysext-work/mounts"
+sysext_base="${sysext_workdir}/base-os.squashfs"
+
+function cleanup() {
+  sudo umount "${sysext_mountdir}"/* || true
+  rm -rf "${sysext_workdir}" || true
+}
+# --
+
+trap cleanup EXIT
+
+rm -rf "${sysext_workdir}" "${sysext_output_dir}"
+mkdir "${sysext_workdir}" "${sysext_output_dir}"
+
+info "creating temporary base OS squashfs"
+sudo mksquashfs "${root_fs_dir}" "${sysext_base}" -noappend
+
+# Build sysexts on top of root fs and mount sysexts' squashfs + pkginfo squashfs
+# for combined overlay later.
+prev_pkginfo=""
+sysext_lowerdirs="${sysext_mountdir}/rootfs-lower"
+for sysext in ${sysexts_list//,/ }; do
+  # format is "<name>:<group>/<package>"
+  name="${sysext%:*}"
+  grp_pkg="${sysext#*:}"
+  create_prod_sysext "${BOARD}" \
+                     "${sysext_output_dir}" \
+                     "${sysext_workdir}" \
+                     "${sysext_base}" \
+                     "${root_fs_dir}"\
+                     "${name}" \
+                     "${grp_pkg}" \
+                     "${prev_pkginfo}"
+
+  mkdir -p "${sysext_mountdir}/${name}" \
+           "${sysext_mountdir}/${name}_pkginfo"
+  sudo mount -rt squashfs -o loop,nodev "${sysext_output_dir}/${name}.raw" \
+                                    "${sysext_mountdir}/${name}"
+  sudo mount -rt squashfs -o loop,nodev "${sysext_output_dir}/${name}_pkginfo.raw" \
+                                    "${sysext_mountdir}/${name}_pkginfo"
+
+  sysext_lowerdirs="${sysext_lowerdirs}:${sysext_mountdir}/${name}"
+  sysext_lowerdirs="${sysext_lowerdirs}:${sysext_mountdir}/${name}_pkginfo"
```

Comment:
It shouldn't matter _here_ because Gentoo tracks installed packages in separate sub-directories (instead of, say, a single file which is appended to), the respective `..._pkginfo`s  only contain `/var/db/pkgs` sub-directories of packages they ship ([build_sysext#L224](https://github.com/flatcar/scripts/blob/contrib/torcx-deprecation-docker-sysext/build_sysext#L224)), and the sysexts themselves only contain their actual binaries in `/usr`.

Bit I'll for sure make a mental note on this, it's important to know.